timemachine/cpp/src/kernels/k_nonbonded.cuh

```diff
@@ -351,6 +351,14 @@ float __device__ __forceinline__ real_es_factor(float real_beta, float dij, floa
     return -inv_d2ij*(static_cast<float>(TWO_OVER_SQRT_PI)*beta_dij*exp_beta_dij_2 + erfc_beta_dij);
 }
 
+float __device__ __forceinline__ fix_nvidia_fmad(float a, float b, float c, float d) {
```

Comment:
I think a comment is worth it here saying that you are fixing issues in the HW fused multiply and add

Reply:
Done!